### PR TITLE
web: disable reading dark mode out of the UI by default

### PR DESCRIPTION
**Pull request by kensternberg-authentik**
## Details

This patch disables “dark mode” as a browser preference. It still honors the user preference, but it will always default to Light mode and will not pay attention to the browser setting.

Thank GNU that dark mode availablity is not a requirement to sell to governments: https://www.section508.gov/content/guide-accessible-web-design-development/

-   [X] The code has been formatted (`make web`)

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -94,8 +94,8 @@
         "@types/grecaptcha": "^3.0.9",
         "@types/guacamole-common-js": "1.5.2",
         "@types/showdown": "^2.0.6",
-        "@typescript-eslint/eslint-plugin": "^7.5.0",
-        "@typescript-eslint/parser": "^7.5.0",
+        "@typescript-eslint/eslint-plugin": "^7.14.0",
```
Comment (kensternberg-authentik) [inline web/package.json:97]: This eliminates the incompatibility warning with Typescript.

File: web/src/elements/Base.ts [hunk 2]
```diff
@@ -77,7 +77,8 @@ export class AKElement extends LitElement {
     }
 
     async getTheme(): Promise<UiThemeEnum> {
-        return rootInterface()?.getTheme() || UiThemeEnum.Automatic;
+        // return rootInterface()?.getTheme() || UiThemeEnum.Automatic;
+        return rootInterface()?.getTheme() || UiThemeEnum.Light;
```
Comment (kensternberg-authentik) [inline web/src/elements/Base.ts:81]: Disabling `Automatic` here means the listener doesn't get installed. The browser is being ignored.  Only the user's explicit setting, retrieved with `getTheme`, will be honored; if it's not there, the default is `Light`.

File: web/src/elements/Base.ts [hunk 3]
```diff
@@ -90,12 +91,9 @@ export class AKElement extends LitElement {
...
         // when dark is preferred
-        this._activateTheme(
-            root,
-            window.matchMedia(QUERY_MEDIA_COLOR_LIGHT).matches
-                ? UiThemeEnum.Light
-                : UiThemeEnum.Dark,
-        );
+        // const pref = window.matchMedia(QUERY_MEDIA_COLOR_LIGHT).matches ? UiThemeEnum.Light : UiThemeEnum.Dark;
+        // this._activateTheme(root, pref);
+        this._activateTheme(root, UiThemeEnum.Light);
```
Comment (kensternberg-authentik) [inline web/src/elements/Base.ts:96]: Always start in `Light` mode; don't read the browser setting.

**Review by kensternberg-authentik — COMMENTED** [review]
Just leaving waypoints.

**Review by fheisler — APPROVED** [review]
part 1 👍

**Pull request merged**